app/forms/additional_claimants_form.rb

```diff
@@ -41,7 +41,8 @@ class ClaimantForm < Form
     validates :title, inclusion: { in: TITLES }, allow_blank: true
     validates :first_name, :last_name, presence: true
     validates :first_name, :last_name, length: { maximum: NAME_LENGTH }
-    validate :older_then_16
+    validate :date_is_valid
```

Comment:
we are already validating the date on line 39 - why do we need to do it again ?  You can always add presence: true to that line ?